ts/features/wallet/cobadge/screen/CobadgeDetailScreen.tsx

```diff
@@ -78,7 +80,23 @@ const CobadgeDetailScreen = (props: Props) => {
     <BasePaymentMethodScreen
       paymentMethod={cobadge}
       card={cardComponent}
-      content={<PaymentMethodFeatures paymentMethod={cobadge} />}
+      content={
+        <Banner
+          pictogramName="feedback"
+          size="big"
+          color="neutral"
+          viewRef={bannerViewRef}
+          title={I18n.t("wallet.methodDetailsWebviewBanner.title")}
+          content={I18n.t("wallet.methodDetailsWebviewBanner.content")}
+          action={I18n.t("wallet.methodDetailsWebviewBanner.cta")}
+          onPress={() =>
+            openAuthenticationSession(
+              "https://io.italia.it/metodi-pagamento",
```

Comment:
Can we move this URL outside the component with a const?

Reply:
addressed 👍

Reply:
I mean, it's a url used in different parts of the app. We should be using a const exported from somewhere. 
Sorry, in the first message it was not clear what I meant!

Reply:
oh well right, addressed this one as well; it most likely is my fault since I was kind of in a hurry 😅